conda/recipes/clx/meta.yaml

```diff
@@ -38,8 +39,8 @@ requirements:
     - pandas
     - python-whois
     - pytest
-    - pytorch 1.3.1
-    - torchvision 0.4.2
+    - pytorch
```

Comment:
Shouldn't you specify the pytorch version here?

Reply:
yes